org.jacktrip.JackTrip.json

```diff
@@ -23,12 +23,19 @@
     ],
     "modules": [
         "pypi-dependencies.json",
+        {
+            "name": "copy-pkgconfig",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p  ${FLATPAK_DEST}/lib/pkgconfig",
+                "cp -nv ${FLATPAK_DEST}/lib/$(gcc --print-multiarch)/pkgconfig/*.pc ${FLATPAK_DEST}/lib/pkgconfig"
```

Comment:
why not setting the pkg-config path is they are not found?

`preprend-pkg-config-path` exist for that.

Reply:
I tried that before and it didn't work locally:
```
"config-opts": [
    "-Dpkg_config_path=/app/lib/$(gcc --print-mutilarch)/pkgconfig"
]
```

Reply:
shell is not substituted in config-opts for meson.

But this is not what I was suggesting.

>        prepend-pkg-config-path (string)
>           This will get prepended to PKG_CONFIG_PATH in the build environment (with an trailing colon if
           needed).

And you can do it per arch.

Reply:
I added `"--prepend-pkg-config-path=/app/lib/$(gcc --print-multiarch)/pkgconfig/"` to finish-args and it doesn't work and several other things. The last thing I've tried was adding the following to the last module:
```
            "arch": {
	    	"x86_64": {
	    		"prepend-pkg-config-path": "/app/lib/x86_64-linux-gnu/pkgconfig/"
	    	}
	    },
```
What is the right thing to do?

Reply:
I think I've got it. Put it under build-options.